manifest/manifest.go

```diff
@@ -118,29 +118,29 @@ func (b *AppUBLock) Show() {
 }
 
 func (m *Manifest) GetUIBlock(id string) *AppUBLock {
-	for _, b := range m.UIBlocks {
+	for i, b := range m.UIBlocks {
 		if b.ID == id {
-			return &b
+			return &m.UIBlocks[i]
```

Comment:
Thank you!

Reply:
You're welcome. As for test coverage, your observation is correct, however, I am doing these changes in the context of the fastlane development of Max Guard. From PoV for the general quality of edge apps a clear micro-framework structure is much more important than a test coverage, which can be provided at any time later.